src/components/sidebar/index.tsx

```diff
@@ -88,12 +88,12 @@ const getFilteredMenuItems = (items: MenuItem[], filterValue: string) => {
   return filteredItems
 }
 
-const Sidebar: React.FC<SidebarProps> = ({
+const Sidebar = ({
+  backToLinkProps = {},
   menuItems,
-  backToLink = {},
   showFilterInput = true,
   title,
-}) => {
+}: SidebarProps): ReactElement => {
```

Comment:
Same case here, using `ReactElement` instead of `React.FC` since this component does not accept a `children` prop.